Luke_Smith_Portfolio_Portfolio.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width initial-scale=1">
+	<title>Luke Smith Portfolio Portfolio</title>
+    <link rel="stylesheet" href="Luke_Smith_Portfolio.css">
+</head>
+<body>
+	<div style="background-color:white;">
+	<div class="topnav">
+    <div class="center">
+		<ul>
+		<li>
+			<a href="Luke_Smith_Portfolio.html">HOME</a>
+		</li>
+	  	<li>
+			<a href="Luke_Smith_Portfolio_About_Me.html">ABOUT ME</a>
+		</li>
+	  	<li>
+			<a href="Luke_Smith_Portfolio_Portfolio.html">PORTFOLIO</a>
+		</li>
+	  	<li>
+			<a href="Luke_Smith_Portfolio_Contact.html">CONTACT</a>
+		</li>
+	    </ul>
+	 </div>
+	</div>
+    <div class="portfolio" style="text-align:center">
+    </div>
+          <img src="Luke_Smith_Logo.png" alt="Luke Logo" style="width:100%">
+    <div class="container" style="text-align:center">
+         <p class="title">PORTFOLIO</p>
+		 <a class="contact" href="file:///Users/lukesmith/Desktop/Formvalidation.html" target="_blank" rel="noopener noreferrer">Project #1</a>
+		 <a class="contact" href="file:///Users/lukesmith/Desktop/pilot-shop/Day5.html" target="_blank" rel="noopener noreferrer">Project #2</a>
+		 <a class="contact" href="file:///Users/lukesmith/Desktop/Day%202.html" target="_blank" rel="noopener noreferrer">Project #3</a>
+	</div>
+    </div>
+</body>
```

Comment:
Your footer is declared outside the body tag. Your footer section should still be enclosed within the main body tag.